### PR TITLE
Added public path static file serving example

### PR DESCRIPTION
**Pull request by johnpyp**
https://github.com/adonisjs/docs/issues/275

### CODE REVIEW

File: 04-http-lifecycle/02-routing.adoc [hunk 1]
```diff
@@ -127,6 +127,14 @@ Route.get('/api/v1/users', 'UserController.index')
 Route.any('*', ({ view }) => view.render('main'))
 ----
 
+You can serve static files, for example the ones generated from create-react-app or vue-cli, like this:
+[source, js]
+----
+const Helpers = use(‘Helpers’)
+// ...
+Route.any('*', ({response}) => response.download(Helpers.publicPath('dist')))
```
Comment (thetutlage) [inline 04-http-lifecycle/02-routing.adoc]: Can you split it in multiple lines. I don't use one liners in docs anywhere

**Comment by johnpyp** [comment]
Is this what you mean?

File: 04-http-lifecycle/02-routing.adoc [hunk 2]
```diff
@@ -127,6 +127,17 @@ Route.get('/api/v1/users', 'UserController.index')
 Route.any('*', ({ view }) => view.render('main'))
 ----
 
+You can serve static files, for example the ones generated from create-react-app or vue-cli, like this:
+[source, js]
+----
+const Helpers = use(‘Helpers’)
+// ...
+Route.any('*', ({response}) => {
```
Comment (RomainLanz) [inline 04-http-lifecycle/02-routing.adoc]: Could you add space between curly brace?

File: 04-http-lifecycle/02-routing.adoc [hunk 3]
```diff
@@ -127,6 +127,17 @@ Route.get('/api/v1/users', 'UserController.index')
...
+  let pathToDist = Helpers.publicPath('dist')
```
Comment (RomainLanz) [inline 04-http-lifecycle/02-routing.adoc]: This should be a const

File: 04-http-lifecycle/02-routing.adoc (hunk 2 above)
Comment (RomainLanz) [inline 04-http-lifecycle/02-routing.adoc]: Could you add space between curly braces?

**Comment by thetutlage** [comment]
Looks 👌

**Pull request merged**